packages/next-swc/crates/next-core/src/next_server/context.rs

```diff
@@ -79,7 +79,10 @@ pub async fn get_server_resolve_options_context(
                 module: true,
                 custom_conditions: vec!["development".to_string()],
                 import_map: Some(next_server_import_map),
-                plugins: vec![external_cjs_modules_plugin.into()],
+                plugins: vec![
+                    external_cjs_modules_plugin.into(),
+                    UnsupportedModulesResolvePluginVc::new(project_path).into(),
```

Comment:
Nit: can we hoist this out and reuse the value?

Reply:
like https://github.com/vercel/next.js/pull/48837/commits/f8809184b1d21d182320b9311069e53ee2e526b5 ? I feel overall moduleoptionscontext in this need a revisit anyway, since there are some repeated codes here and there.